mlir/lib/Dialect/Affine/Transforms/SuperVectorize.cpp

```diff
@@ -1223,8 +1223,19 @@ static Operation *vectorizeAffineLoad(AffineLoadOp loadOp,
   LLVM_DEBUG(dbgs() << "\n[early-vect]+++++ permutationMap: ");
   LLVM_DEBUG(permutationMap.print(dbgs()));
 
+  // Make sure that the in_bounds attribute corresponding to a broadcast dim
+  // is set to `true` - that's required by the xfer Op.
```

Comment:
Nothing for this PR but this is still a constraint that seems unnecessary and makes this complicated, as we can see here.

Reply:
Agreed. Expect a PR to remove this shortly :) (~next week)